multinode_integration_tests/src/main.rs

```diff
@@ -489,6 +592,125 @@ mod tests {
         assert_eq!(data_hunk.data, vec!(1, 2, 3, 4));
     }
 
+    #[test]
+    fn can_read_probe_target_from_str() {
+        probe_target_pairs()
+            .into_iter()
+            .for_each(|(string, expected)| {
```

Comment:
You probably should name the local variable `string` better. Perhaps `literal_target` or `str_target_address`.

Even the word `expected` is a bit vague, lacking a good context. I know you can argue that the test name can lead you around down to the idea but there might be a way to write it more expressively.

Reply:
Fixed